docs/semgrep-code/editor.md

```diff
@@ -165,12 +166,16 @@ Adding a rule to the Policies page applies the rule across all projects scanned
 To add a rule to the Policies page:
 
 1. Click **Add to Policy**.
-2. Select which rule mode (Monitor, Comment, or Block) to set the rule to, depending on the rule’s relevance.
+2. Select which [rule mode](docs/semgrep-code/policies/#rule-pane-in-detail) (Monitor, Comment, or Block) to set the rule to, depending on the rule’s relevance.
```

Comment:
This is where I want to refer to the rule modes externally.

Reply:
I think the #rule-pane-in-detail section should be broken into tables with headings, so that we can reference it. I'm pretty sure that's what i did for the (now gone) rule-board, and I did it for [ssc triage and remediation](https://semgrep.dev/docs/semgrep-supply-chain/triage-and-remediation/) :) lmk if you want me to take over this PR?

Reply:
Yes - if the changes I've made so far seem reasonable, please feel free to do that bit and merge!